raiden/tests/unit/test_pfs_integration.py

```diff
@@ -153,6 +158,10 @@ def test_get_pfs_info_success():
     with patch.object(requests, 'get', return_value=response):
         pathfinding_service_info = get_pfs_info("url")
 
+        assert pathfinding_service_info['price_info'] == 0
+        assert pathfinding_service_info['network_info']['chain_id'] == 1
+        assert pathfinding_service_info['network_info']['registry_address'] == \
```

Comment:
style: We don't split lines with `\` in the codebase. IF needed to split it then do:

either of:

```python
registry_address = pathfinding_service_info['network_info']['registry_address']
assert registry_address ==  '0xB9633dd9a9a71F22C933bF121d7a22008f66B908'
```

```python
assert pathfinding_service_info['network_info']['registry_address'] == (
    '0xB9633dd9a9a71F22C933bF121d7a22008f66B908'
)
```

```

Reply:
thanks, done